README.md

```diff
@@ -17,17 +17,17 @@ enabled by default in the GCE cluster.
 There are tons of node problems that could possibly affect the pods running on the
 node, such as:
 * Infrastructure daemon issues: ntp service down;
-* Hardware issues: Bad cpu, memory or disk;
+* Hardware issues: Bad CPU, memory, or disk;
 * Kernel issues: Kernel deadlock, corrupted file system;
 * Container runtime issues: Unresponsive runtime daemon;
 * ...
 
-Currently these problems are invisible to the upstream layers in cluster management
+Currently, these problems are invisible to the upstream layers in the cluster management
```

Comment:
Why is this necessary?

Reply:
https://www.grammarly.com/blog/commas-after-introductory-words/